src/index.js

```diff
@@ -32,9 +33,10 @@ var Time = React.createClass({
     var machineReadable = value.format('YYYY-MM-DDTHH:mm:ssZ');
 
     if (relative || format) {
+      var titleValue = value.format(titleFormat || 'YYYY-MM-DD HH:mm');
       var humanReadable = relative ? value.fromNow() : value.format(format);
       return (
-        <time {...props} dateTime={machineReadable}>
+        <time {...props} dateTime={machineReadable} title={ relative ? titleValue : null }>
```

Comment:
I once forced myself to write idiomatic.js and now it's so ingrained... Will fix.